openssl/src/ssl/mod.rs

```diff
@@ -3083,6 +3093,40 @@ impl SslRef {
         }
     }
 
+    /// Returns the stack of ciphers from the ciphers field of the client's hello message, and
+    /// the signalling cipher suite values.
+    ///
+    /// This can only be used inside of the client hello callback. Otherwise, `None` is returned.
+    ///
+    /// Requires OpenSSL 1.1.1 or newer.
+    #[corresponds(SSL_bytes_to_cipher_list)]
+    #[cfg(ossl111)]
+    pub fn client_hello_ciphers_stack(&self) -> Option<(Stack<SslCipher>, Stack<SslCipher>)> {
+        unsafe {
+            let mut ptr = ptr::null();
+            let len = ffi::SSL_client_hello_get0_ciphers(self.as_ptr(), &mut ptr);
```

Comment:
I generally prefer to bind to each OpenSSL API as directly as possible, so I think we'd want separate methods for SSL_client_hello_get0_ciphers, SSL_bytes_to_cipher list, and SSL_client_hello_isv2 (though really the answer to that is just `false` as this point!).

Reply:
In that case as `SSL_client_hello_get0_ciphers` and `SSL_client_hello_isv2` both already have bindings, would you accept a new method like the following?
```rust
// struct Ssl
#[corresponds(SSL_bytes_to_cipher_list)]
pub fn bytes_to_ciphers_stack(&self, bytes: &[u8], isv2format: bool) -> Option<(Stack<SslCipher>, Stack<SslCipher>)> { // ... }
```